packages/react-mutation-mapper/src/util/FilterUtils.ts

```diff
@@ -17,6 +17,26 @@ import { ApplyFilterFn } from '../model/FilterApplier';
 
 export const TEXT_INPUT_FILTER_ID = '_mutationTableTextInputFilter_';
 
+export function filteredDataByCodon(
```

Comment:
We don't need to recompute filtered data here. This data already exists in the mutation mapper store. We just need a `@computed` value to get `samplesByPosition`.

Another thing is the `sample` defined in this function is actually a `Partial<Mutation>`, we can probably use `mutationsByPosition` as is without even adding a new function.

Reply:
This function is removed now.